docs/target.settings.yaml

```diff
@@ -4,7 +4,18 @@ general:
   cm_device_codename: grouper
   cm_release_type: snapshot
   cm_release_version: M12
-  parent_template: mia-default
+  parent_templates:
+    - mia-default
+    - wifi-only
```

Comment:
ok, and just to make sure we're on the same page: if `wifi-only` has a `settings.yaml` with `mia-default` as *its* only entry in `parent_templates` (as I was intending it to), then we don't explicitly need to add both here (as it should recurse in and find it anyhow), but you're just doing that to demonstrate the we *can* is we want?